src/codeu/chat/server/View.java

```diff
@@ -48,7 +48,9 @@ public View(Model model) {
     this.model = model;
   }
 
-
+  public ServerInfo getInfo(){
+    return null;
```

Comment:
Why does this return null? Seems like a good place for a comment, or maybe an `UnsupportedOperationException` if this isn't supposed to be called ever.